core/lib/libefi/include/efi/mpinfo.h

```diff
@@ -0,0 +1,136 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2024, Arm Limited and Contributors. All rights reserved.
+ *
+ * * @par Reference(s):
+ * - UEFI Platform Initialization Specification
+ *   (https://uefi.org/specs/PI/1.8/index.html)
+ */
+
+#ifndef __MPINFO_H
+#define __MPINFO_H
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <util.h>
+
+/*
+ * Value used in the NumberProcessors parameter of
+ * the GetProcessorInfo function
+ */
+#define CPU_V2_EXTENDED_TOPOLOGY  UL(1 << 24)
```

Comment:
Please use one of the `BIT()` or `BIT64()` macros.

Reply:
Thanks!